test/read.js

```diff
@@ -52,7 +167,7 @@ describe("Testing jsonapi-client", function() {
       var somePhoto = person.photos[0];
       assert.throws(function() {
         somePhoto.relationships("photographer").add("foobar");
-      }, /Expected Resource, got String/);
+      }, /Expected Resource, got /);
```

Comment:
Got it: https://travis-ci.org/holidayextras/jsonapi-client/jobs/104183675
```
IE 9.0.0 (Windows 7 0.0.0) LOG: 'foobar', undefined
.........
IE 9.0.0 (Windows 7 0.0.0) Testing jsonapi-client fetches foreign resources throws an error when adding a non-resource FAILED
	Expected Resource, got undefined
.......
```
```
IE 11.0.0 (Windows 10 0.0.0) LOG: 'foobar', undefined
.........
IE 11.0.0 (Windows 10 0.0.0) Testing jsonapi-client fetches foreign resources throws an error when adding a non-resource FAILED
	Error: Expected Resource, got undefined
	   at Resource.prototype._checkIsResource (/home/travis/build/holidayextras/jsonapi-client/dist/jsonapi-client-test.js:138:3)
	   at Resource.prototype._addRelation (/home/travis/build/holidayextras/jsonapi-client/dist/jsonapi-client-test.js:242:3)
	   at add (/home/travis/build/holidayextras/jsonapi-client/dist/jsonapi-client-test.js:231:7)
	   at Anonymous function (/home/travis/build/holidayextras/jsonapi-client/dist/jsonapi-client-test.js:16781:9)
	   at _throws (/home/travis/build/holidayextras/jsonapi-client/dist/jsonapi-client-test.js:850:5)
	   at assert.throws (/home/travis/build/holidayextras/jsonapi-client/dist/jsonapi-client-test.js:876:3)
	   at Anonymous function (/home/travis/build/holidayextras/jsonapi-client/dist/jsonapi-client-test.js:16780:7)
.......
```
and:
```javascript
  var type = typeof resource;
  if (resource.constructor) type = resource.constructor.name;
console.log(resource, type); // <-- here's the output above
  throw new Error("Expected Resource, got " + type);
```